ada/testsuite/tests/ada_api/static_expr_eval/main.adb

```diff
@@ -9,17 +10,25 @@ procedure Main is
    Unit : Analysis_Unit := Get_From_File (Ctx, "test.adb");
 
    function Is_Object_Decl (N : Ada_Node) return Boolean
-   is (Kind (N) in Ada_Object_Decl);
+   is (Kind (N) in Ada_Object_Decl
+         and then not N.As_Object_Decl.F_Default_Expr.Is_Null);
 begin
    for E of Find (Root (Unit), Is_Object_Decl'Access).Consume loop
-      declare
-         Res : Eval_Result := Expr_Eval (E.As_Object_Decl.F_Default_Expr);
       begin
-         Put_Line ("Expr " & Short_Image (E) & " evaluated to " & Image (Res));
-         if Res.Kind in Int | Enum_Lit then
-            Put_Line ("   Int value is " & As_Int (Res).Image);
-         end if;
-         Put_Line ("");
+         declare
+            Res : Eval_Result := Expr_Eval (E.As_Object_Decl.F_Default_Expr);
+         begin
+            Put_Line 
+               ("Expr " & Short_Image (E) & " evaluated to " & Image (Res));
+            if Res.Kind in Int | Enum_Lit then
+               Put_Line ("   Int value is " & As_Int (Res).Image);
+            end if;
+            Put_Line ("");
```

Comment:
I know it’s not new, but it would be nice if you could replace this call to `Put_Line` with a call to `New_Line` (same below).